app/src/main/java/org/ometa/lovemonster/service/ResponseParser.java

```diff
@@ -196,6 +196,12 @@ private Calendar parseDatetime(@Nullable final String datetime) {
      *      the non-blank value, or null otherwise
      */
     private String parseString(final JSONObject jsonObject, final String attributeName) {
+        // JSONObject will helpfully coerce null to "null" for us.  We don't want that.
+        // http://code.google.com/p/android/issues/detail?id=13830
+        if (jsonObject.isNull(attributeName)) {
```

Comment:
Hmm, that's strange.  I was definitely seeing messages of "null" in the app.

I'll try reproducing in a test later this weekend.